tests/end_to_end_tests/trainer_test.py

```diff
@@ -37,61 +39,39 @@ def tearDownClass(cls) -> None:
     def get_classification_trainer(name=''):
         trainer = Trainer(name, model_checkpoints_location='local')
         dataset_params = {"batch_size": 4}
-        dataset = ClassificationTestDatasetInterface(dataset_params=dataset_params)
+        dataset = ClassificationTestDatasetInterface(dataset_params=dataset_params, image_size=224)
```

Comment:
why was that changed?

Reply:
We do not run the e2e in the ci atm, it was broken before this change.